.rubocop.yml

```diff
@@ -1,5 +1,22 @@
+require:
+  - rubocop-rspec
+
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 3.1
+  SuggestExtensions: false
+  DisplayCopNames: true
+  Include:
```

Comment:
Can you remove includes/excludes for files that don't exist in this repo?

Reply:
I guess I was thinking that the boilerplate is harmless enough, and we do have rake files ...

Reply:
For me, boilerplate is fine in files that we never look at, but counts as cruft when in files that we regularly touch/think about.

Reply:
removed!